npc/quests/bard_quest.txt

```diff
@@ -1925,9 +1925,9 @@ morocc,134,111,3	script	Bard#3	2_M_BARD_ORIENT,{
 				mes "You don't need to be scared, that person is a servant of God, okay?";
 			} else if (Class == Job_Thief || Class == Job_Rogue) {
 				mes "There's no reason to be afraid of this riffraff, your Uncle Kino is here, okay?";
-			} else if (BaseClass == Job_Assassin) {
+			} else if (Class == Job_Assassin) {
 				mes "There's no reason to be afraid. I know that person looks scary, but you're a good girl, so you'll be okay.";
-			} else if (BaseClass == Job_Blacksmith) {
+			} else if (Class == Job_Blacksmith || (BaseJob == Job_Blacksmith && Upper == 2)) {
```

Comment:
this condition satisfy the following jobs
* Job_Blacksmith 
* Job_Baby_Blacksmith
* Job_Baby_Mechanic
doesn't satisfy these job
* Job_Mechanic
* Job_Whitesmith

something must be wrong here,
why Job_Baby_Mechanic a baby 3rd job can satisfy this condition while normal 3rd job can't ?